package.json

```diff
@@ -117,5 +121,8 @@
   ],
   "engines": {
     "node": "^12.20.0 || ^14.14.0 || >=16.0.0"
+  },
+  "ava": {
+    "verbose": true
```

Comment:
That was previously specified inside the ava config file shouldn't we add the files here as well?

```
export default {
  files: ['test/*.js'],
  verbose: true,
}
```

Reply:
`files` finds `test/*.js` [by default](https://github.com/avajs/ava/blob/main/docs/05-command-line.md) in Ava, so we might be able to remove it there.

Reply:
This will be moot in ava v4 since only the verbose reporter is left IIRC :)